js/main.js

```diff
@@ -202,14 +210,50 @@ Site.Programacion = {
     bindToggle: function() {
       var _this = this;
 
-      $('.programacion-drawer-toggle').on('click', function() {
+      $('.programacion-title, .programacion-drawer-toggle, .close-drawer-overlay').bind('click', function() {
         _this.toggleContent();
       });
+
+      $('#programacion-slider').on({
+        mousewheel: function(event){
+          var delta = event.originalEvent.wheelDelta;
+
+          if(delta < 0 && !$('.programacion-drawer').hasClass('drawer-open')){
```

Comment:
there needs to be overflow in the panel. and this is the interaction they asked for